class.jetpack-heartbeat.php

```diff
@@ -113,6 +113,7 @@ public static function generate_stats_array( $prefix = '' ) {
 		$return["{$prefix}language"]       = get_bloginfo( 'language' );
 		$return["{$prefix}charset"]        = get_bloginfo( 'charset' );
 		$return["{$prefix}is-multisite"]   = is_multisite() ? 'multisite' : 'singlesite';
+		$return["{$prefix}is-https"]       = is_ssl() ? 'https' : 'http';
```

Comment:
Maybe put this next to the `ssl` one, so we can see at a glance that they're tracking different things. I like adding it though -- would be nice to see when we reach a tipping point on HTTPS support.